index.js

```diff
@@ -14,37 +14,40 @@ const { getAuth } = require("./util");
  *   password: string
  * }} */
 const cfg = context.configuration;
+const allowAggregations = [
+  "AVERAGE",
+  "AVG",
+  "COUNT",
+  "COUNT_DISTINCT",
+  "COUNT_UNIQUE",
+  "LENGTH",
+  "MAX",
+  "MIN",
+  "SORTED_UNIQUE",
+  "STDDEV",
+  "STDDEV_POPULATION",
+  "STDDEV_SAMPLE",
+  "SUM",
+  "UNIQUE",
+  "VARIANCE",
+  "VARIANCE_POPULATION",
+  "VARIANCE_SAMPLE"
+];
 
-if (
-  ![
-    "LENGTH",
-    "MIN",
-    "MAX",
-    "SUM",
-    "AVERAGE",
-    "STDDEV_POPULATION",
-    "STDDEV_SAMPLE",
-    "VARIANCE_POPULATION",
-    "VARIANCE_SAMPLE",
-    "UNIQUE",
-    "SORTED_UNIQUE",
-    "COUNT_DISTINCT",
-    "COUNT",
-    "AVG",
-    "STDDEV",
-    "VARIANCE",
-    "COUNT_UNIQUE"
-  ].includes(cfg.aggregation.toUpperCase())
-) {
+const AGG_NAME = cfg.aggregation.toUpperCase();
+
+if (!allowAggregations.includes(AGG_NAME)) {
+  const allowed = allowAggregations.join(", ");
   throw new Error(
     `Invalid service configuration. Unknown aggregation function: ${
       cfg.aggregation
-    }`
+    }, allow are ${allowed}`
```

Comment:
I think this is meant to read "allowed are" but in any case just inlining the array results in icky formatting.

Maybe try this:
```js
`Invalid service configuration. Unknown aggregation function: ${
  cfg.aggregation
}. Please use one of: ${allowed.join(", ")}.`
```

Reply:
done